src/components/expanded-state/unique-token/UnderlyingAsset.js

```diff
@@ -39,7 +38,7 @@ export default function UnderlyingAsset({
     const asset =
       ethereumUtils.getAccountAsset(address) ||
       ethereumUtils.formatGenericAsset(
-        genericAssets[toLower(address)],
+        genericAssets[address.toLowerCase()],
```

Comment:
```suggestion
        genericAssets[address?.toLowerCase()],
```

Reply:
I rechecked it and didn't run into any issues. In many places, I added extra optional chaining that can be removed after full coverage with typescript.
I also could not get null or undefined in the allocated place. However, I will add this change and, as I said above, with full coverage of the typescript, it will be possible to remove the redundant code